tests/phpstan-baseline.neon

```diff
@@ -0,0 +1,49 @@
+parameters:
+	ignoreErrors:
+		-
+			message: '#^Cannot call method matches\(\) on Hamcrest\\Matcher\|null\.$#'
+			identifier: method.nonObject
+			count: 1
+			path: ../hamcrest/Hamcrest/Arrays/SeriesMatchingOnce.php
+
+		-
+			message: '#^Parameter \#1 \$matcher of method Hamcrest\\Arrays\\SeriesMatchingOnce\:\:_describeMismatch\(\) expects Hamcrest\\Matcher, Hamcrest\\Matcher\|null given\.$#'
+			identifier: argument.type
+			count: 1
+			path: ../hamcrest/Hamcrest/Arrays/SeriesMatchingOnce.php
+
+		-
+			message: '#^Cannot call method toString\(\) on class\-string\|object\.$#'
+			identifier: method.nonObject
+			count: 1
+			path: ../hamcrest/Hamcrest/Core/HasToString.php
+
+		-
+			message: '#^Cannot cast Hamcrest\\BaseDescription to string\.$#'
+			identifier: cast.string
+			count: 1
+			path: ../hamcrest/Hamcrest/StringDescription.php
+
+		-
+			message: '#^Argument of an invalid type list\<string\>\|false supplied for foreach, only iterables are supported\.$#'
+			identifier: foreach.nonIterable
+			count: 1
+			path: ../hamcrest/Hamcrest/Text/IsEqualIgnoringWhiteSpace.php
+
+		-
+			message: '#^Cannot access offset int\<0, max\> on non\-empty\-array\<int\<0, max\>, string\>\|false\.$#'
+			identifier: offsetAccess.nonOffsetAccessible
+			count: 1
+			path: ../hamcrest/Hamcrest/Text/IsEqualIgnoringWhiteSpace.php
+
+		-
+			message: '#^Method Hamcrest\\Xml\\HasXPath\:\:matchesContent\(\) has parameter \$nodes with generic class DOMNodeList but does not specify its types\: TNode$#'
+			identifier: missingType.generics
+			count: 1
+			path: ../hamcrest/Hamcrest/Xml/HasXPath.php
```

Comment:
no idea what needs to put in there.

Btw: ext-dom is missing from the composer.json, do you want to add it?
and ext-ctype is used and missing, too

Reply:
> no idea what needs to put in there.

Same here :( Let's look at this later.

> Btw: ext-dom is missing from the composer.json, do you want to add it?
and ext-ctype is used and missing, too

Please do this in a separate PR, because it's unrelated.

Reply:
Will be fixed separately as part of the https://github.com/hamcrest/hamcrest-php/issues/99.